bigquery/tests/unit/test_job.py

```diff
@@ -2720,6 +2720,89 @@ def test_reload_w_alternate_client(self):
         self.assertEqual(req['path'], PATH)
         self._verifyResourceProperties(job, RESOURCE)
 
+    def test_to_dataframe(self):
+        import pandas as pd
+
+        begun_resource = self._makeResource()
```

Comment:
**NOTE**: Method names should still adhere to PEP8 in unit tests. (I'll send a PR to rename `_makeResource` as `_make_resource`, so don't worry about it in this PR.) (Sorry our tooling didn't complain about this.)

Reply:
@alixhami You'll want to rebase after #4355 was merged

Reply:
Rebased and updated the function name in ea0c219